### PR TITLE
[Improvement] Improve server module

### PR DESCRIPTION
**Pull request by s7monk**
### Purpose

Improve tablecontroller

### CODE REVIEW

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/controller/TableController.java [hunk 1]
```diff
@@ -269,22 +270,30 @@ private List<TableChange> createTableChanges(TableColumn oldColumn, TableColumn
...
-     *     exception occurs, returns a result object with an error status.
+     * @param catalogName the name of the catalog where the database is located.
+     * @param databaseName the name of the database where the table is located.
+     * @param tableName the name of the table to which options are added.
+     * @param options a map of options to add to the table.
+     * @return a response indicating the success or failure of the operation.
      */
     @PostMapping("/option/add")
-    public R<Void> addOption(@RequestBody TableInfo tableInfo) {
+    public R<Void> addOption(
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/controller/TableController.java:280]: Pls revert this change. It's better to use `tableInfo`.

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/data/model/DatabaseInfo.java [hunk 2]
```diff
@@ -32,8 +32,6 @@ public class DatabaseInfo {
 
     private String databaseName;
 
-    private Integer catalogId;
```
Comment (SteNicholas) [inline paimon-web-server/src/main/java/org/apache/paimon/web/server/data/model/DatabaseInfo.java:35]: Why remove this id?

**Review by SteNicholas — COMMENTED** [review]
@s7monk, thanks for the contribution. I left some comments. PTAL.

File: paimon-web-server/src/main/java/org/apache/paimon/web/server/data/model/DatabaseInfo.java (hunk 2 above)
Comment (s7monk) [reply paimon-web-server/src/main/java/org/apache/paimon/web/server/data/model/DatabaseInfo.java:35]: > Why remove this id?

catalogId is a useless field here. Paimon's catalog, database, table, etc. do not have id field information. If this change is unnecessary, I will close this PR.
